spec/models/organization_spec.rb

```diff
@@ -8,15 +8,44 @@
   it { should validate_uniqueness_of(:login) }
 
   describe 'load_organizations' do
+    let(:whitelisted_organisations) { ['test-organization'] }
+    before do
+      stub_const("Organization::WHITELISTED_ORGS", whitelisted_organisations)
+    end
+
     context 'creates new organization' do
-      let(:whitelisted_organisations) { ['test-organization'] }
+      it 'create a new Organization record' do
+        expect { Organization.load_organizations }.to change{ Organization.count }.by(1)
+      end
+    end
 
+    context 'updates organization' do
+      it 'updates fields if they were changed' do
+        Organization.load_organizations
+        organization = Organization.find_by_login('test-organization')
+        organization.name = 'old_name'
+        organization.github_url = 'https://github.com/old_organization_url'
+        organization.save!
+        Organization.load_organizations
+        updated_organization = Organization.find_by_login('test-organization')
+
+        expect{ organization.reload }.to change { organization.name }
+                                           .from('old_name')
+                                           .to(updated_organization.name)
+                                           .and change { organization.github_url }
+                                                  .from('https://github.com/old_organization_url')
+                                                  .to(updated_organization.github_url)
+      end
+    end
+
+    context 'call LoadRepositoriesJob after create organization' do
       before do
-        stub_const("Organization::WHITELISTED_ORGS", whitelisted_organisations)
+        allow(LoadRepositoriesJob).to receive(:perform_later)
+        Organization.load_organizations
       end
 
-      it 'create a new Organization record' do
-        expect { Organization.load_organizations }.to change{ Organization.count }.by(1)
+      it 'call LoadRepositoriesJob' do
+        expect(LoadRepositoriesJob).to have_received(:perform_later)
```

Comment:
I don't like this. Can you please check that you have `LoadRepositoriesJob` queued after the `load_organizations`?
Instead you're nailing down your implementation details in your tests — it is a bad practice, since it prevents you from any meaningful refactoring later.

(Rule of thumb: check _effects_ of your actions, not the _actions_ themselves. The effect you desire here is "having jobs enqueued", you shouldn't care about _which method call_ gets you there, if that avoidable)